dev/com.ibm.ws.microprofile.reactive.messaging_fat/publish/servers/ProducerRecordMessagingServer/bootstrap.properties

```diff
@@ -0,0 +1,14 @@
+###############################################################################
+# Copyright (c) 2017, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+bootstrap.include=../testports.properties
+osgi.console=5471
+com.ibm.ws.logging.trace.specification = *=info:logservice=detail:REACTIVEMESSAGE=all:com.ibm.ws.microprofile.reactive.messaging.kafka.adapter.impl.*=FINEST
+use.kafka.producer.record=true
```

Comment:
Comment to point out that this is a beta flag (and also TODO to remove later)

Reply:
Done